ArraysHomework/ArraysHomework.cs

```diff
@@ -127,6 +153,9 @@ public void Lists()
             // Swap the last and first value of this list.
             List<string> swapMyValues = new List<string> { "first", "middle", "last" };
 
+            swapMyValues[0] = swapMyValues[2];
```

Comment:
This is close. But `swapMyValues[0] = swapMyValues[2]` changes the value at index 0, so the original first element is lost.
How can you hold onto a copy of the original value before changing it?